db/migrate/20201219120055_add_order_to_adjustments.rb

```diff
@@ -0,0 +1,18 @@
+class AddOrderToAdjustments < ActiveRecord::Migration
+  def up
+    add_column :spree_adjustments, :order_id, :integer
+
+    # Ensure migration can use the new column
+    Spree::Adjustment.reset_column_information
+
+    # Migrate adjustments on orders
+    Spree::Adjustment.where(order_id: nil, adjustable_type: "Spree::Order").find_each do |adjustment|
```

Comment:
so we plan to drop this kind of adjustable in favor of `order_id`?

Reply:
It will become more common for an adjustment to be associated more clearly with a specific object, like a line_item or a shipment, but all adjustments still ultimately relate to a specific order.

Reply:
Hopefully it'll become a bit clearer in the next couple of PRs after this one, where we'll add tax_categories to shipping_methods, allowing shipments to have proper tax adjustments for different tax rates (currently they don't).

It then becomes useful to have nice scopes where you can sum (for example) all adjustments on all items (line items or shipments) related to an order.

Reply:
the only thing that worries me is the redundancy of the association to an order. Either directly through order_id or indirectly through shipment or line_item. I guess we can fix that later on